mne/time_frequency/psd.py

```diff
@@ -80,12 +80,21 @@ def psd_array_welch(x, sfreq, fmin=0, fmax=np.inf, n_fft=256, n_overlap=0,
         The length of FFT used, must be ``>= n_per_seg`` (default: 256).
         The segments will be zero-padded if ``n_fft > n_per_seg``.
     n_overlap : int
-        The number of points of overlap between segments. Will be adjusted
-        to be <= n_per_seg. The default value is 0.
+        The number of points of overlap between segments. Must be < n_per_seg.
+        The default value is 0.
     n_per_seg : int | None
         Length of each Welch segment. The smaller it is with respect to the
         signal length the smoother are the PSDs. Defaults to None, which sets
         n_per_seg equal to n_fft.
+    reduction : str | float | callable | None
+        The type of reduction to perform on windows. If string it can be 'mean'
+        or 'median'. If float it is understood as % of values to trim before
+        performing mean (has to be > 0 and < 0.5) ie. trimmed-mean. If function
+        it has to perform the reduction along last dimension. If None - no
+        reduction is performed and psd's for individual windows are returned
+        (so that the output psds are of shape (windows, channels, frequencies)
+        for 2d input data and (windows, epochs, channels, frequencies) for 3d
+        input data)
```

Comment:
Oh, that's actually wrong. You previously commented that it should be -2 dim and I changed it that way. It makes sense as -2 - especially for epochs `epochs x windows x channels x freqs`)